run_analysis.py

```diff
@@ -154,6 +142,22 @@ def analysis(config): #{{{
     config = MpasAnalysisConfigParser()
     config.read(configFileNames)
 
+    # choose the right rendering backend, depending on whether we're displaying
+    # to the screen
+    if not config.getboolean('plot', 'displayToScreen'):
+        mpl.use('Agg')
+    import matplotlib.pyplot as plt
+    # analysis can only be imported after the right MPL renderer is selected
+    from mpas_analysis.ocean.ohc_timeseries import ohc_timeseries
+    from mpas_analysis.ocean.sst_timeseries import sst_timeseries
+    # from mpas_analysis.ocean.nino34_timeseries import nino34_timeseries
+    # from mpas_analysis.ocean.mht_timeseries import mht_timeseries
+    # from mpas_analysis.ocean.moc_timeseries import moc_timeseries
+    from mpas_analysis.ocean.ocean_modelvsobs import ocn_modelvsobs
+
+    from mpas_analysis.sea_ice.timeseries import seaice_timeseries
+    from mpas_analysis.sea_ice.modelvsobs import seaice_modelvsobs
+
```

Comment:
@pwolfram, I guess once the imports can't go at the top of the file, it doesn't matter much how you break that convention.  I prefer having them together in `__main__` so they happen as early as possible and so they are in scope for any functions that would need them.  But that's not a practical reason at this point since we don't have any functions other than `analysis(...)`.  The other thing is that it's not desirable to have multiple imports of the same thing within and if statement when multiple analyses use the same function (e.g. `ocean.modelvsobs`) but this doesn't seem to have caused a problem.

In general, what I have read is that every effort should be made in python not to do local imports e.g. within funcitons, so I was doing my best to avoid that approach.

I'm sympathetic to the idea that it should be a simple as possible for future users to add new scripts, so keeping everything in one place as it was before would probably be best.  I'll change it back.